packages/create-sitecore-jss/src/templates/angular/sitecore/definitions/config.js

```diff
@@ -0,0 +1,8 @@
+// this file is imported by default prior to executing the jss manifest command
+// use this to enable transpilation or any other pre-manifest configurations that are needed.
+
+console.log('Enabling TypeScript transpilation for the manifest...');
+
+require('ts-node').register({
+	project: 'src/tsconfig.webpack-server.json'
```

Comment:
Fix indent, I know that originally it was as it is, but let's fix while we are working on it